umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md

```diff
@@ -63,21 +53,14 @@ The [Kafka Broker](/docs/architecture/microservices/core/kafka-broker/) uses the
 
 Follow these steps to access the Kafka Broker outside the cluster:
 
-1. Open UMHLens / OpenLens and navigate to **Network > Services**.
-2. Select the Service `united-manufacturing-hub-kafka-external` and click the **Edit** button.
-3. Scroll down to the line that shows:
-
-   ```yaml
-     type: NodePort
-   ```
+1. Access your instance via SSh
+2. Execute this command:
 
-   Replace this with:
-
-   ```yaml
-     type: LoadBalancer
+   ```bash
```

Comment:
Can we add a test that will test whether this command will work? similar to the ones to get credentials?